### PR TITLE
DetailsList: Update Check sizing so padding is included in button rather than parent element

### PR DESCRIPTION
**Pull request by leddie24**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ npm run change`

#### Description of changes

Updates check so padding is included in parent ```<button>``` so hit target is more accessible to mobile users.

Screenshot of new touch target for button:
(Black border is the new touch target)
screen shot 2017-09-12 at 10 36 08 am

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/DetailsList/DetailsRowCheck.scss [hunk 1]
```diff
@@ -18,9 +18,9 @@
 }
 
 button.check {
-  height: 20px;
-  width: 20px;
-  padding: 0;
+  height: 100%;
+  width: 40px;
+  padding: 6px 8px;
```
Comment (dzearing) [inline packages/office-ui-fabric-react/src/components/DetailsList/DetailsRowCheck.scss]: Please don't use padding like this ever for things that are supposed to be fixed. Why 6px? Why 8? I think you are making an assumption about the size of the contents of the button... As soon as a designer changes the size of the check, the button will.

Instead, you should center the content within.

File: packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.scss [hunk 2]
```diff
@@ -53,7 +53,7 @@ $isPaddedMargin: 24px;
 
   &.cellIsCheck {
     position: relative;
-    padding: 6px 8px;
+    padding: 0;
```
Comment (dzearing) [inline packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.scss]: Shouldn't this have a height somewhere?

**Comment by dzearing** [comment]
Check the focus on the DetailsHeader select all button.

File: packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.scss (hunk 2 above)
Comment (dzearing) [reply packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.scss]: Can you set focus to the select all button and see the focus rect?

Comment (leddie24) [reply packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.scss]: Height is set in .cell (32px).  Should the DetailsHeader be 40px then?

Comment (dzearing) [reply packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.scss]: no probably not. But it should be the same height as the other cells in the header I believe.

**Pull request merged**